lib/SIL/SILVerifier.cpp

```diff
@@ -2083,40 +2083,41 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
             "Operand of " #name "_to_ref does not have the " \
             "operand's type as its referent type"); \
   }
-#define ALWAYS_LOADABLE_CHECKED_REF_STORAGE_HELPER(Name, name, closure) \
```

Comment:
I am not sure why clang-format reformatted this. I can undo this hunk if you want.